Assets/Project/Scripts/GamePlayScene/Bottle/DynamicBottleController.cs

```diff
@@ -21,6 +21,25 @@ public class DynamicBottleController : AbstractBottleController
         /// </summary>
         [SerializeField] protected AnimationClip warpReverseAnimation;
 
+        /// <summary>
+        /// 動くことができる状態か
+        /// </summary>
+        private bool _isMovable = true;
+
+        public bool IsMovable
+        {
+            get => _isMovable;
+            set {
+                if (value) {
```

Comment:
```suggestion
                if (value == _isMovable) return;
                if (value) {
```
じゃないと連続 `true` に設定した場合、 ハンドラーが複数登録されてしまう

Reply:
69c3904